e2e/e2e-windows-scenario.sh

```diff
@@ -45,6 +44,36 @@ collect-logs() {
     echo "collect cse logs done"
 }
 
+DEPLOYMENT_VMSS_NAME="$(mktemp -u winXXXXX | tr '[:upper:]' '[:lower:]')"
+export DEPLOYMENT_VMSS_NAME
+
+# download azcopy
+wget https://aka.ms/downloadazcopy-v10-linux
+tar -xvf downloadazcopy-v10-linux
+
+# zip and upload cse package
+cd ../staging/cse/windows
+zip -r ../../../e2e/${DEPLOYMENT_VMSS_NAME}-aks-windows-cse-scripts.zip ./* -x ./*.tests.ps1 -x "*azurecnifunc.tests.suites*" -x README -x provisioningscripts/*.md -x debug/update-scripts.ps1
```

Comment:
It is better to add timestamp in the package name so we can cleanup all old packages when it fails in deleting the uploaded package

Reply:
Maybe use timestamp together with the random-generated vmss name to name cse package is better?